Packs/AtlassianConfluenceCloud/Integrations/AtlassianConfluenceCloud/AtlassianConfluenceCloud.py

```diff
@@ -1333,14 +1379,101 @@ def confluence_cloud_group_list_command(client: Client, args: Dict[str, str]) ->
         raw_response=response_json)
 
 
+def fetch_events(client: Client, last_run: dict[str, Any], fetch_limit: int):
+    demisto.debug(f'Starting fetch_events with last_run: {last_run} and fetch_limit: {fetch_limit}')
+    end_date = int((time.time() - 5) * 1000)
+    last_end_date = last_run.get('end_date', 0)
+    start_date = last_end_date + 1 if last_end_date else end_date - 60000
```

Comment:
this is an hour back by default? lets make it 2 minutes and extract to a global constant

Reply:
It's a minute, end_date is in milliseconds not seconds. done